app/src/main/java/org/p2p/wallet/sell/ui/payload/SellPayloadPresenter.kt

```diff
@@ -78,6 +79,7 @@ class SellPayloadPresenter(
     }
 
     private suspend fun checkForSellLock() {
+        needCheckForSellLock = false
```

Comment:
What does it mean? 
Sell Lock? Does it mean the user is blocked for selling?
Maybe you'll call it `isSellAvailable` then?

Reply:
we have a fragment `SellLocked` when the user is locked from the sell 
so we check that is Sell is locked for user, naming is ok here, i approve